cli/types/cypress.d.ts

```diff
@@ -2352,9 +2352,14 @@ declare namespace Cypress {
      * @default "spec"
      */
     reporter: string
+    /**
+     * Some reporters accept [reporterOptions](https://docs.cypress.io/guides/tooling/reporters.html#Reporter-Options) that customize their behavior
+     * @default "spec"
+     */
+    reporterOptions: { [key: string]: any }
     /**
      * Whether to take a screenshot on test failure when running headlessly or in CI
```

Comment:
Old comment leftover from copy/paste. 

```suggestion
     * Customize reporter behavior by passing options
```